### DISCUSSION TITLE
Is it possible for packagers to customize the DefaultVideoSavePath on linux?

### DISCUSSION THREAD

**Discussion by timthelion** [obsproject/obs-studio#5896] [Development Questions] [answered]
I see that in x11 the DefaultVideoSavePath is set to $HOME.
https://github.com/obsproject/obs-studio/blob/076cd5d5d4794dced38676e4ecf8d1fc9d97e692/UI/platform-x11.cpp#L211

I need to change that to fix https://github.com/snapcrafters/obs-studio/issues/65
Is there already an ENV var to do this? Or should I add a new one? I would name it something like: `$OBS_STUDIO_DEFAULT_VIDEO_SAVE_PATH`.

**Comment by kkartaltepe** [comment, marked as answer]
Would https://github.com/obsproject/obs-studio/pull/3746 resolve your issue?

**Reply by timthelion** [reply]
This would indeed be workable. A little more complex but perfectly workable I think.
